CMakeLists.txt

```diff
@@ -32,12 +32,12 @@ SET(PROJECT_DESCRIPTION "${PROJECT_NAME_UCASE} - Free music production software"
 SET(PROJECT_COPYRIGHT   "2008-${PROJECT_YEAR} ${PROJECT_AUTHOR}")
 SET(VERSION_MAJOR       "1")
 SET(VERSION_MINOR       "1")
-SET(VERSION_PATCH       "91")
-#SET(VERSION_SUFFIX     "")
-SET(VERSION             "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}")
-IF(VERSION_SUFFIX)
-	SET(VERSION     "${VERSION}-${VERSION_SUFFIX}")
-ENDIF(VERSION_SUFFIX)
+SET(VERSION_BUILD       "91")
```

Comment:
No, I mean as part of this commit, you should also bump our internal version to use the new convention, `1.2.0-rc2`, etc as `1.1.91` was only being used before we had proper `-rcX`, `-alphax` parsing logic.